package.json

```diff
@@ -35,7 +35,8 @@
     "get-prefix": "^1.0.0",
     "react": "^0.14.0",
     "react-measure": "^0.3.4",
-    "react-motion": "^0.4.2"
+    "react-motion": "^0.4.2",
+    "react-prop-types": "^0.3.0"
```

Comment:
@frederickfogerty not the best with prod builds, but I'm guessing this will screw up webpack dist builds if the end user doesn't have that lib? Can we make it so it includes whatever we grab from react-prop-types?